misc/scripts/deploy-tools.sh

```diff
@@ -210,6 +211,8 @@ function deployESXInstall {
 }
 
 function deployESXPost {
+    # restart hostd so that esxcli extension file is read
+    $SSH $TARGET $HOSTD restart
```

Comment:
I'm not sure its ok to restart hostd when the VIB is installed. It may be disruptive. If this is the typical  practice then I guess thats fine. Just checking.